packages/tinacms/src/plugins/fields/ColorFieldPlugin.tsx

```diff
@@ -42,4 +43,5 @@ export const ColorField = wrapFieldsWithMeta<InputProps, ColorFieldProps>(
 export default {
   name: 'color',
   Component: ColorField,
+  parseString,
```

Comment:
Looks good! The name of the property being exported should be `parse` though.

```suggestion
  parse: parseString
```

Reply:
Ah ok, cool!

Reply:
These changes have been made in the latest commit